stock_picking_eco_tag/report/stock_picking_eco_tag_template.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="label_transfer_template_eco_view_pdf">
+        <t t-call="web.basic_layout">
+            <div class="page">
+                <t t-foreach="docs" t-as="picking">
+                    <t t-foreach="picking.move_lines" t-as="move">
+                        <t t-foreach="move.move_line_ids" t-as="move_line">
+                          <div style="display: inline-table; height: 160px; width: 367px; border: 1px solid black;">
+                            <div class="d-flex">
+                                <div style="display: -webkit-box; display: -webkit-flex; display: flex; -webkit-box-align: center; justify-content: center; align-items: center; margin-left: 3px; width: 35%">
+                                    <img t-attf-src="data:image/jpg;base64,{{ move.company_id.logo }}" style="width: 130px"/>
+                                </div>
+                                <div style="width: 65%; padding-left: 6px;" t-if="picking.move_lines">
+                                    <div style="padding-top: 5px;">
+                                        <p class="mb-0" style="font-size: 10px">Product: <t t-esc="move_line.product_id.name"/></p>
+                                        <p class="mb-0" style="font-size: 10px">Var / Cat: <t t-esc="move_line.product_id.categ_id.name[3:]"/></p>
+                                        <p class="mb-0" style="font-size: 10px">Net weight: <t t-esc="move_line.qty_done"/> <t t-esc="move_line.product_id.uom_id.name"/></p>
```

Comment:
Missing numer format. A custom format related to user or picking partner should be required. @lmiguens-solvos 
- how many decimals should we need?
- lang format, related to Odoo user, or picking partner?

Reply:
- We need 3 decimals. But this precision maybe from self.env['decimal.precision'].precision_get('Product Unit of Measure')
- About the language, I'm not sure, but maybe it should be the company language or picking partner... but it is possible tags should always go in one language?

Reply:
> * We need 3 decimals. But this precision maybe from self.env['decimal.precision'].precision_get('Product Unit of Measure')

I think it should be possible. @ChristianSantamaria could you try?

> * About the language, I'm not sure, but maybe it should be the company language or picking partner... but it is possible tags should always go in one language?

It's possible creating a `res.config.settings` parameter, with an extra stuff. If you prefer it rather than the other options, that shoud be the way IMO.

Reply:
On second thought ... perhaps the best may be the language of the picking.partner, right? 
So whoever receives the label, understand it (assuming it is generated in the picking of the sales order)

Reply:
@dalonsod Yes of course, i start now

Reply:
Then,
- quantity number format according picking partner language.
- decimal precision according @lmiguens-solvos requirements